packages/apps/plugins/plugin-space/package.json

```diff
@@ -65,9 +66,12 @@
     "@types/react-dom": "^18.0.6",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "vite": "^5.0.12"
+    "vite": "^5.0.12",
+    "vue": "^3.3.12",
+    "vuepress": "2.0.0-rc.0"
```

Comment:
hmmm, skeptical about all the changes here. What prompted the inclusion of vuepress? Whatever that was we should fix that because this definitely shouldn't be here.

Reply:
whoops, was a leftover from some issues `vuepress-theme-hope` was causing unalignment with the 2.0.0-rc.X versions, which I solved by un-locking `@vuepress/plugin-search`. not sure if we want to look at bumping all of those at some point?